python/pyproject.toml

```diff
@@ -20,7 +20,7 @@ dependencies = [
 
 [project.optional-dependencies]
 srt = ["aiohttp", "fastapi", "psutil", "rpyc", "torch", "uvloop", "uvicorn",
-       "zmq", "vllm>=0.3.3", "interegular", "pydantic", "pillow", "outlines>=0.0.27"]
+       "zmq", "vllm==0.3.3", "interegular", "pydantic", "pillow", "outlines>=0.0.27"]
```

Comment:
Recently, we moved to vllm==0.4.2, can you update accordingly?
There are only minor changes. You can change similar to the changes in this PR https://github.com/sgl-project/sglang/pull/380